CONTRIBUTING.md

```diff
@@ -151,15 +151,15 @@ If you would like to work on something in the admin panel (which includes some p
 
 To enable access to the panel:
 * Go to your database, and find the `mship_account_role` table. Set the `role_id` to `1` for your CID.
-* Navigate to `vukcore.localhost/adm/dashboard`. You should now be able to log into the admin panel. 
+* Navigate to `vukcore.localhost/adm/dashboard`. You should now be able to log into the admin panel.
 
 #### Relax
 
 After all that setup, relax for 5 minutes!  If you've had any problems, come and find someone in the Slack team.
 
 ## Contributing to the code
 
-If you're just getting started with GitHub (and project contributions) then we suggest you take a look at issues marked with the "up-for-grabs" label.  These issues will be of reasonable size and challenge, for anyone to start contributing to the project.  [This was inspired by an article by Kent C. Dodds](https://medium.com/@kentcdodds/first-timers-only-78281ea47455#.wior7p101).
+If you're just getting started with GitHub (and project contributions) then we suggest you take a look at issues [marked with the "up-for-grabs" label](https://vatsimuk.atlassian.net/browse/CORE-26?jql=project%20%3D%20CORE%20AND%20labels%20%3D%20up-for-grabs%20order%20by%20lastViewed%20DESC).  These issues will be of reasonable size and challenge, for anyone to start contributing to the project.  [This was inspired by an article by Kent C. Dodds](https://medium.com/@kentcdodds/first-timers-only-78281ea47455#.wior7p101).
```

Comment:
Will we be masking the domain with a CNAME? If so, can we update the link to reflect that please?

Reply:
Custom domains and CNAMEs are not currently supported by Atlassian's Cloud Offerings. I have spoken to @NFarrington about implementing some basic sub-domain rewrite rules to redirect users to the site - not sure how far that is out so I would say it is probably best to leave them as is for now and when we implement the rewrites I can quickly update then.

What do you think? @CalumTowers

Reply:
@daviesnathan Disappointing to learn that enterprise grade software won't allow white labelling... We should keep an eye on https://jira.atlassian.com/browse/CLOUD-6999.

Reply:
Completely agree, I have voted on it to try and bump it up and I am actively watching it. Apparently they have it in a Private Beta at the moment so shouldn't be too long, the rewrites along with parameter passing should give a similar experience in the meantime.